.github/workflows/find_unused.yml

```diff
@@ -0,0 +1,13 @@
+name: Find unused files
+on: push
```

Comment:
This might not need to be done on every push, since there are unlikely to be a lot of these picked up. Can we make it less frequent?

Reply:
No, it should be on every push, to make sure we never have any stale file.

@juliogonzalez Do we have a limit on GH actions minutes?

Reply:
@paususe https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration

Reply:
I think we need to seriously consider the issue of GH actions minutes, especially since we are also implementing l10n and linting actions. I would prefer to spend those minutes on actions that will have a higher positive rate.